src/icons.js

```diff
@@ -6,7 +6,5 @@ module.exports = {
     forbidden: String.fromCodePoint(9940),
     no: String.fromCodePoint(10060),
     yes: String.fromCodePoint(9989),
-    left: String.fromCodePoint(11013),
-    right: String.fromCodePoint(10145),
     eyes: String.fromCodePoint(128064),
-}
```

Comment:
I think it would be better to keep those here even if they aren't used at the moment